back/src/domains/convention/use-cases/notifications/NotifyActorThatConventionNeedsModifications.ts

```diff
@@ -187,10 +187,11 @@ const requesterNameByRole = (
     "establishment-representative": `${convention.signatories.establishmentRepresentative.firstName} ${convention.signatories.establishmentRepresentative.lastName} (le représentant légal de l'entreprise)`,
     counsellor: agency.name,
     validator: agency.name,
-    "establishment-tutor": new Error(wrongRequesterUser),
-    toReview: new Error(wrongRequesterUser),
     backOffice: "L'équipe Immerssion Facilité",
```

Comment:
```suggestion
    backOffice: "L'équipe Immersion Facilitée",
```

C'est pas dans le code concerné par la PR, mais tant qu'on y est